test/unit-test/README.md

```diff
@@ -0,0 +1,18 @@
+# Unit Tests
+
+To add or update unit tests, follow the instructions from the [development guide](https://escomp.github.io/CAM-SIMA-docs/atmospheric_physics/development_workflow/#5-unit-testing).  Also, make sure [pFUnit](https://github.com/Goddard-Fortran-Ecosystem/pFUnit) is built and installed following the [build directions](https://github.com/Goddard-Fortran-Ecosystem/pFUnit?tab=readme-ov-file#building-and-installing-pfunit) (see the associated workflow file for a detailed example).
+
+To run the tests, from the root directory of your clone, run:
+
```

Comment:
Can we include instructions on building pFUnit?
```
git clone --depth 1 --branch v4.10.0 https://github.com/Goddard-Fortran-Ecosystem/pFUnit.git
cd pFUnit
pwd
cmake -B./build -S.
cd build
make install
```

Reply:
Added!

There's no mechanism to inject text from another file at the moment but I might be able to make a small shell script with this and then the docs can expand that in place and be used by the github workflow but that's a change for another day.

Reply:
also, not to harp on this, but the steps to run the test are explicitly listed here even though they're also in the workflow. Is it that much worse to include the explicit instructions for how to build pFUnit here?

Reply:
Added a link to the workflow file!

I think we want to keep our duplication of instructions to a minimum as (at least in this case), this script is going to change and there's a risk they will diverge at some point.

It's not going to be difficult to fix or triage if we notice something incorrect later, it's just an additional location of documentation we need to be aware of as we make changes going forward.